src/components/TextLink/index.tsx

```diff
@@ -0,0 +1,46 @@
+/** @jsx jsx */
+import * as React from "react";
+import { jsx } from "@emotion/core";
+import Text, { FontSizes } from "components/Text";
+import { Theme } from "types/global";
+
+interface Props {
+  href?: string;
+  size?: FontSizes;
+}
+
+const TextLink: React.FC<Props> = ({ ...props }) => (
+  <Text
+    css={({ ...theme }: Theme) => ({
+      // we are using an 8 digit hex code to generate the boxShadow opacity
+      boxShadow: `0px 1px 0 ${theme.colors.dark}30`,
+      color: "inherit",
+      cursor: "pointer",
+      fontSize: props.size ? "" : "inherit",
+      textDecoration: "none",
+      position: "relative",
+      "&:after": {
+        background: theme.colors.dark,
+        bottom: -1,
+        content: "''",
```

Comment:
is `"''"` intentional?

Reply:
Yeah it is necessary to set the value to ''